tests/bsim/bluetooth/host/misc/hfc/src/main.c

```diff
@@ -0,0 +1,445 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdio.h>
+
+#include <zephyr/kernel.h>
+#include <zephyr/sys/byteorder.h>
+#include <zephyr/bluetooth/bluetooth.h>
+#include <zephyr/bluetooth/hci.h>
+#include <zephyr/bluetooth/l2cap.h>
+#include <zephyr/bluetooth/att.h>
+#include <zephyr/bluetooth/gatt.h>
+#include <zephyr/bluetooth/uuid.h>
+#include <zephyr/bluetooth/conn.h>
+
+#include "testlib/att_read.h"
+
+#include <argparse.h>		/* For get_device_nbr() */
```

Comment:
get_device_nbr is nowhere is this PR, is the comment outdated and maybe this include not needed?